src/AstariaRouter.sol

```diff
@@ -575,7 +575,10 @@ contract AstariaRouter is
         ILienToken.InvalidLienStates.AMOUNT_ZERO
       );
     }
-    if (newStack.lien.details.duration < s.minLoanDuration) {
+    if (
+      newStack.lien.details.duration < s.minLoanDuration ||
+      newStack.lien.details.duration == 0
```

Comment:
if we enforce that `s.minLoanDuration > 0` (in the initialiser or when filing updates) the second check `newStack.lien.details.duration == 0` is not neccessary (since it will be implied)

Reply:
Yes this was for a case where we fail to set the min duration properly. Now or in any future updates.